src/medVtkInria/vtkDataManagement/vtkMetaDataSetSequence.cxx

```diff
@@ -854,9 +848,7 @@ void vtkMetaDataSetSequence::ComputeSequenceDuration()
         return;
     }
 
-    double step = (this->GetMaxTime() - this->GetMinTime())/(double)(this->GetNumberOfMetaDataSets()-1);
-
-    this->SequenceDuration = this->GetMaxTime() - this->GetMinTime() + step;
+    this->SequenceDuration = this->GetMaxTime() - this->GetMinTime();
```

Comment:
there was a problem with this getMaxTime as far as i remember, which gave the last sequence time without its duration, that's why we added "step". Is it ok now? i'm a bit lost with these PRs

Reply:
GetMaxTime is supposed to give the latest time found in all the data sets of the sequence.
If there was a problem, it must have come from the data sets themselves.

To me this class has a problem with how time is handled, but this is outside the scope of this PR.

Reply:
(this line was not modified by us, except for indentation, and dates back to the original medInria)

This is discussion is similar to the one we had at https://github.com/Inria-Asclepios/medInria-public/pull/257#issuecomment-367291453 in which I agree ```getMaxTime``` works as it should.

 I also share your view @mjuhoor that the way time sequences are handled is faulty. They have timestamps for each frame but the code assumes that the frame duration is fixed, which is bad design (if the frame duration is fixed, you don't need timestamps, just the frame duration). Having timestamps allows sequences of irregular frame duration, except most of the code doesn't handle that correctly. While reviewing that PR I wanted to go further into those issues but after searching the code I found the problem stems all the way to DTK and would require many changes (so yes, as you say, it is outside the scope of this PR). As a consequence, I think we should stay with the assumption that we have fixed-duration frames (important for the next paragraph).

The code changed here (which dates back to medInria, not us) might work better now, but there is still the problem of the last step. Each timestamp marks the beginning of a frame, so the last frame's timestamp is not the end of the sequence. The last frame starts at ```getMaxTime``` but ends later. This means the total duration was correctly given before. I fear changing this will cause the export of looping videos to have a minor frame jump at the end.

Reply:
(But really if this is too problematic to fix properly then I suppose we should just take the version that doesn't cause visual artefacts, which, from your first comment, seems to be your version)